.github/assets/css/vendor.css

```
@@ -0,0 +1,50 @@
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed,
+figure, figcaption, footer, header, hgroup,
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
```

Biome (1.9.4)

[error] Unexpected shorthand property font after font-size



(lint/suspicious/noShorthandPropertyOverrides)